frontend/src/routes/(frontend)/explore/index.tsx

```diff
@@ -44,13 +44,10 @@ export default component$(() => {
 	)
 })
 
-export const requestLoader = loader$(async ({ request }) => {
-	// Manually parse the JSON to ensure that Qwik finds the resulting objects serializable.
-	return JSON.parse(JSON.stringify(request)) as Request
-})
+export const requestUrlLoader = loader$(async ({ request: { url } }) => url)
```

Comment:
NIT - I find this easier to grok... is there any reason for it to be destructured in the parameter list?

```suggestion
export const requestUrlLoader = loader$(async ({ request }) => request.url)
```

Reply:
no not really I thought it looked neater, but I don't mind either version 🙂